contrib/plugins/cowsay.sh

```diff
@@ -0,0 +1,25 @@
+#! /bin/sh
+
+# Eg. {"jsonrpc":"2.0","id":2,"method":"getmanifest","params":{}}\n\n
+read JSON
+read BLANK
```

Comment:
Since `BLANK` is intentionally unused: consider renaming it `_` to make that clear. Applies to all `BLANK`:s.